python/fichas/listas.py

```diff
@@ -99,7 +99,7 @@ def maximum(values: list[int]) -> int:
     return result
 
 
-def sums(values: list[int]) -> list[int]:
+def sums(values: list) -> list:
```

Comment:
I think we should also keep the types for list items. We could make it more generic and not int specific, but that would require importing more thinks until we get python 3.10. 

See some sugestions [here](https://github.com/muesli/duf). Although I think make it an int, works fine.